cmd/kubeadm/app/phases/upgrade/compute_test.go

```diff
@@ -61,14 +62,20 @@ func (f *fakeVersionGetter) KubeletVersions() (map[string]uint16, error) {
 	}, nil
 }
 
-type fakeEtcdCluster struct{}
+type fakeEtcdCluster struct{ TLS bool }
```

Comment:
Could you please open an issue about instead of creating a stub to use the test jiggery that exists to auto-standup and cleanup etcd.